clients/cobol-lsp-vscode-extension/src/__tests__/LanguageClientServiceTest.spec.ts

```diff
@@ -104,4 +104,12 @@ describe("LanguageClientService positive scenario", () => {
             },
         });
     });
+
+    test("Test LanguageClientService exit language server is calls exit on languageClient", () => {
```

Comment:
It is not clear to me what does this test checks from the description

Reply:
updated